### DISCUSSION TITLE
Code Owner Bypass Option for Rulesets

### DISCUSSION THREAD

**Discussion by Morpheus636** [community/community#68271] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

When configuring rulesets, an option that allows codeowners to bypass the ruleset would be really useful for me. For example, I want to make it so that review is required for non-codeowners to modify files that have codeowners, but codeowners should be able to modify anything they own without review. 

If anyone has a better way to do this (GitHub actions are an option), let me know!

**Comment by marcaurele** [comment]
I think the implementation of the rules by GH does not let you do it unless they review the meaning of the bypass group as described in my ticket: https://github.com/orgs/community/discussions/74831
